src/desktop/apple_notification.mm

```diff
@@ -47,40 +32,20 @@ bool available() {
 	return false;
 }
 
-#ifdef HAVE_NS_USER_NOTIFICATION
 void send_cocoa_notification(const std::string& owner, const std::string& message);
-#endif
-#ifdef HAVE_GROWL
-void send_growl_notification(const std::string& owner, const std::string& message, const desktop::notifications::type note_type);
-#endif
 
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunused-parameter"
-#ifdef HAVE_NS_USER_NOTIFICATION
 void send_notification(const std::string& owner, const std::string& message, const desktop::notifications::type note_type) {
     @autoreleasepool {
         Class appleNotificationClass = NSClassFromString(@"NSUserNotificationCenter");
         if (appleNotificationClass) {
```

Comment:
As you're also dropping 10.7 support, this test is no longer needed (it's testing to confirm the OS has the NSUserNotificationCenter API and 10.8 and later always have that).

Reply:
Sure, that's why I complete removed `HAVE_NS_USER_NOTIFICATION` macro

Reply:
The Macro confirms the compiler has support for the class (it prevents compile errors). This is the matching runtime test, which can also be eliminated once the binary no longer runs on 10.7